.travis.yml

```diff
@@ -1,6 +1,5 @@
 language: node_js
 node_js:
-  - 6
```

Comment:
dont we run node v.6.12.2 on cominor?

Reply:
Ya I think we do, im not sure if we use something like `nave` to run cimpler in a different version

Reply:
Shoot, I thought we were like on 10+, I need to double-check

Reply:
```
root@ubreakit.com  14:05:16 ~ > node --version  
v6.12.2                                         
```
:cry:

Reply:
Double checking that we're using `nave` for cimpler...

Reply:
Nope! We're running on the system node for cimpler still :sweat: 
https://github.com/iFixit/server-templates/blob/54d82e2ef1b1b16ddfbb043966bf68eab732be07/specific/dev/startup-scripts/attachments/cimpler.service#L7

Reply:
I'd be fine using `nave`, a couple of our internal test nodes use it already (testbeacon, jest)